core_integrations/esera_onewire

```diff
@@ -0,0 +1 @@
+../core_brands/esera
```

Comment:
There aren't multiple esera integrations, thus brands should not be used. Please create a regular integration entry.

Reply:
Done